lib/src/builders/message/message.dart

```diff
@@ -0,0 +1,232 @@
+import 'package:nyxx/src/builders/builder.dart';
+import 'package:nyxx/src/builders/message/allowed_mentions.dart';
+import 'package:nyxx/src/builders/message/attachment.dart';
+import 'package:nyxx/src/builders/sentinels.dart';
+import 'package:nyxx/src/models/message/embed.dart';
+import 'package:nyxx/src/models/message/message.dart';
+import 'package:nyxx/src/models/snowflake.dart';
+
+class MessageBuilder extends CreateBuilder<Message> {
+  final String? content;
+
+  final dynamic /* int | String */ nonce;
+
+  final bool? tts;
+
+  final List<Embed>? embeds;
+
+  final AllowedMentions? allowedMentions;
+
+  final Snowflake? replyId;
+
+  final bool? requireReplyToExist;
+
+  // TODO
+  //final List<ComponentBuilder>?> components;
```

Comment:
when these gonna be done? Also should we move part of interactions library to main framework like Message builder stuff for creating messages with components and needed data structure? All logic for handling interactions should still be in _interactions package but data and parsing part would be better here